src/Components/ItemProperties.jsx

```diff
@@ -342,28 +348,48 @@ const dms = (deg, min, sec) => {
 
 const useStyles = makeStyles({
   propsContainer: {
+    '& h2, & h3, p': {
+      margin: '0.25em 0',
+      padding: 0,
+    },
+    '& h2': {
+      fontSize: '1.1em',
+      fontWeight: 800,
```

Comment:
this seems to much - keep it at 600 - maybe.

Reply:
Yeah, gave it a try but doesn't go between.. it's either thin or heavy, at least for me...  give this a try.